tensorflow/python/compiler/tensorrt/trt_convert_test.py

```diff
@@ -70,7 +70,8 @@ class TrtConvertTest(test_util.TensorFlowTestCase, parameterized.TestCase):
 
   # Use a small max_workspace_size for tests so they don't consume too much GPU
   # memory.
-  _TRT_MAX_WORKSPACE_SIZE_BYTES = trt_convert.DEFAULT_TRT_MAX_WORKSPACE_SIZE_BYTES
+  _TRT_MAX_WORKSPACE_SIZE_BYTES = \
```

Comment:
No need to backslash \. Please remove.

Reply:
I'm not sure to understand your suggestion. In Python you need the backslash for line breaks.

Reply:
Sorry for not being clear, yes, python style guide says:
Do not use backslash line continuation except for with statements requiring three or more context managers.